lib/helper/user_comms.rb

```diff
@@ -0,0 +1,25 @@
+module Emerald
+  class UserCommsHelper
+
+    ERROR_INCORRECT_FILE_TYPE = "Oops! Incorrect file type received.\nExpecting a lisp file e.g. example.lisp\n"
+    ERROR_INITIALISE_WITH_STRING_IO = "Error: Initialize with StringIO objects"
+
+    def initialize(stdout)
+      raise ERROR_INITIALISE_WITH_STRING_IO if stdout.nil?
+      @stdout = stdout if stdout.respond_to?(:puts)
```

Comment:
Hmm ok, yeah feels a bit unsafe but I suppose it depends on how you feel about the trade off. Since the above scenario is unlikely to happen, I've removed the check this time.